pkg/controller/volume/scheduling/metrics/metrics.go

```diff
@@ -40,7 +40,7 @@ var (
 		&metrics.HistogramOpts{
 			Subsystem:      VolumeSchedulerSubsystem,
 			Name:           "scheduling_duration_seconds",
-			Help:           "Volume scheduling stage latency",
+			Help:           "Volume scheduling stage latency (Deprecated since 1.18.0)",
```

Comment:
nit: would be great to add it to other metrics than: 

https://github.com/kubernetes/kubernetes/blob/5b3fe0564d27feb7a744c23856d31eddbe6b70ea/pkg/scheduler/metrics/metrics.go#L108-L117

https://github.com/kubernetes/kubernetes/blob/5b3fe0564d27feb7a744c23856d31eddbe6b70ea/pkg/scheduler/metrics/metrics.go#L98-L107


From changelog:

> The scheduling_duration_seconds summary metric is deprecated (#86586, @xiaoanyunfei) [SIG Scheduling]
The scheduling_algorithm_predicate_evaluation_seconds and scheduling_algorithm_priority_evaluation_seconds metrics are deprecated, replaced by framework_extension_point_duration_seconds[extension_point="Filter"] and framework_extension_point_duration_seconds[extension_point="Score"]. (#86584, @xiaoanyunfei) [SIG Scheduling]

Reply:
Thank you for your comments. I'll add them also. Are these metrics deprecated since 1.19.0? They are mentioned in CHANGELOG-1.18.md.